src/main/webapp/resources/js/components/Header/MainNavigation/MainNavigation.css

```diff
@@ -39,7 +39,7 @@
 
 .main-navigation .ant-menu-title-content a {
   font-weight: 600;
-  font-size: 1.5rem;
+  font-size: 1.2rem;
```

Comment:
Why are we reducing the main nav font size here?

Reply:
When I removed bootstrap, this font in the nav blew up.  This just put it back in check

Reply:
Ok weird, glad to know that it is the same size pre PR